### ISSUE TITLE
get credentials from provider's  installed app

### ISSUE THREAD

**Issue by fly-lover** [oauth-io/oauth-phonegap#46]
Hello!
Is it possible to get user information from installed facebook,twitter or whatever application? 
and if not installed, from inappbrowser

**Comment by calclavia** [comment]
Yeah, I'm also trying to look for a solution to this. Possibly one could use Cordova FB plugin, and then hook it into OAuth somehow...

**Comment by angelxmoreno** [comment]
You will need to use some sort of Facebook or Twitter cordova/phonegap plugin that will allow you that can of access.

**Comment by calclavia** [comment]
Yes. Then how do we create a request object from the tokens obtained by those plugins?

**Comment by angelxmoreno** [comment]
Looks like you can. [According to the docs](http://docs.oauth.io/?iOS#request-objects) you can create object that can be used to create a request object
